cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/ordered/AbstractTable.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright © 2014 Cask Data, Inc.
```

Comment:
Copyright should be 2015. Please check other classes, too. Have to updated your IntelliJ settings?